src/tests/work-item/work-item-list/assigne.spec.js

```diff
@@ -0,0 +1,149 @@
+      });
+    });
+  }); 
+ /**User can Cancel assignee  */
+   it('User can Cancel assignee -phone ', function() {
+      var workItemTitle = "The test workitem title";
+      var workItemUpdatedTitle = "The test workitem title - UPDATED";
+      page.clickWorkItemQuickAdd();
+      page.typeQuickAddWorkItemTitle(workItemTitle);
+      page.clickQuickAddSave().then(function() {
+      page.workItemViewId(page.workItemByTitle(workItemTitle)).getText().then(function (text) {
+      var detailPage = page.clickWorkItemTitle(page.firstWorkItem, text);
+      detailPage.workItemDetailAssigneeIcon().click();
+      detailPage.setWorkItemDetailAssigneeSearch("Harry Potter",false);
+      detailPage.clickAssignedUserDropDownList("Harry Potter");
+      expect(detailPage.details_assigned_user().getText()).toContain("Harry Potter");
+      detailPage.details_assigned_user().click();
+      detailPage.clickworkItemDetailCancelButton();
+      expect(detailPage.details_assigned_user().getText()).toContain("Harry Potter");
+      });
+    });
+  }); 
+
+  /**User can read , update , remove assignee  */
+    it('User can read , update , remove assignee -phone ', function() {
+      var workItemTitle = "The test workitem title";
+      var workItemUpdatedTitle = "The test workitem title - UPDATED";
+      page.clickWorkItemQuickAdd();
+      page.typeQuickAddWorkItemTitle(workItemTitle);
+      page.clickQuickAddSave().then(function() {
+      page.workItemViewId(page.workItemByTitle(workItemTitle)).getText().then(function (text) {
+      var detailPage = page.clickWorkItemTitle(page.firstWorkItem, text);
+      detailPage.workItemDetailAssigneeIcon().click();
+      detailPage.setWorkItemDetailAssigneeSearch("Harry Potter",false);
+      detailPage.clickAssignedUserDropDownList("Harry Potter");
+      expect(detailPage.details_assigned_user().getText()).toContain("Harry Potter");
+      detailPage.details_assigned_user().click();
+      detailPage.setWorkItemDetailAssigneeSearch("Walter Mitty",false);
+      detailPage.clickAssignedUserDropDownList("Walter Mitty");
+      detailPage.details_assigned_user().click();
+      detailPage.clickworkItemDetailCancelButton();
+      expect(detailPage.details_assigned_user().getText()).toContain("Walter Mitty");
+      });
+    });
+  });
+  /**Test name and avatar are shown up in the drop down */ 
+  it('Test name and avatar are shown up in the drop down -phone ', function() {
+      page.workItemViewId(page.firstWorkItem).getText().then(function (text) { 
+      var detailPage = page.clickWorkItemTitle(page.firstWorkItem, text);
+      detailPage.workItemDetailAssigneeIcon().click();
+      detailPage.setWorkItemDetailAssigneeSearch("Walter Mitty",false);
+      detailPage.clickAssignedUserDropDownList("Walter Mitty");
+      expect(detailPage.details_assigned_user().getText()).toContain("Walter Mitty");
+      expect(detailPage.workItemDetailAvatar().isPresent()).toBeTruthy();
```

Comment:
I know what you're getting at here, but I think we need to make a change. 

For an unassigned user, the css is:  .pull-left.fa.fa-user-plus.user-assign-icon
For an assigned user, the css is: .pull-left.user-assign-avatar